frontend/src/components/Landing/PsychologistFinder.jsx

```diff
@@ -18,16 +19,13 @@ const PsychologistFinder = () => {
           Contacter un psychologue partenaire dans n‘importe quel département,
           peu importe votre université d‘origine, par téléphone, par email ou par son site web.
```

Comment:
```suggestion
          Contacter un psychologue partenaire quel que soit son département et
          votre université d‘origine, par téléphone, par email ou par son site web.
```
Retour wording de Lina

Reply:
```suggestion
          Je contacte un psychologue partenaire quel que soit son département et
          mon université d‘origine, par téléphone, par email ou par son site web.
```
2ème proposition de wording de Lina (en gardant le même ton que les 3 blocs en haut de page "Je consulte un médecin..."